### PR TITLE
Arrange mmetrics code and doc

### PR DESCRIPTION
**Pull request by cryeo**
- Arrange code and docs in `mmetrics.R`

### CODE REVIEW

**Comment by yoshinaga0106** [comment]
Thank you for your arrange!!
I also tried to check. However, the below error occurred in add method. Is this error occured in your environment?
スクリーンショット 2019-07-25 10 08 27

File: R/mmetrics.R [hunk 1]
```diff
@@ -57,50 +59,48 @@ define <- function(...){
...
 #' @export
-gsummarize <- function(df, ..., metrics){
-  group_vars <- rlang::enquos(...)
-  metrics <- mfilter(metrics, df)
-  df %>%
-    dplyr::group_by(!!!group_vars) %>%
-    dplyr::summarise(!!!metrics) %>%
-    dplyr::ungroup()
-}
+gsummarize <- function(df, ..., metrics) dplyr::summarise %>% gprocess(df, ..., metrics = metrics)
```
Comment (shinichi-takayanagi) [inline R/mmetrics.R]: We (in R) usually use `%>%` operator for `data.frame` object for easy understanding of the data flow.
That is why we hardly use `%>%` operator for function.

Could you write simply like `gprocess(dplyr::summarise, df, ..., metrics = metrics)` ?

File: R/mmetrics.R [hunk 2]
```diff
@@ -57,50 +59,48 @@ define <- function(...){
...
 #' @export
-gmutate <- function(df, ..., metrics){
-  group_vars <- rlang::enquos(...)
-  metrics <- mfilter(metrics, df)
-  df %>%
-    dplyr::group_by(!!!group_vars) %>%
-    dplyr::mutate(!!!metrics) %>%
-    dplyr::ungroup()
-}
+gmutate <- function(df, ..., metrics) dplyr::mutate %>% gprocess(df, ..., metrics = metrics)
```
Comment (shinichi-takayanagi) [inline R/mmetrics.R]: The same with https://github.com/y-bar/mmetrics/pull/49#discussion_r307105245

File: R/mmetrics.R [hunk 3]
```diff
@@ -57,50 +59,48 @@ define <- function(...){
...
-  metrics <- mfilter(metrics, df)
-  df %>%
-    dplyr::group_by(!!!group_vars) %>%
-    dplyr::summarise(!!!metrics) %>%
-    dplyr::ungroup()
-}
+gsummarize <- function(df, ..., metrics) dplyr::summarise %>% gprocess(df, ..., metrics = metrics)
+
 #' @rdname add
+#' @export
```
Comment (shinichi-takayanagi) [inline R/mmetrics.R:74]: +1

**Pull request merged**